client/src/components/propertyhome/PropertyHome.js

```diff
@@ -52,6 +70,10 @@ function PropertyHome() {
           </div>
           <div className='tenants-view-container border-left'>
             <TenantView propertyId={property._id} />
+            <section className='sectionContainer'>
+            {tenantAddIsOpen && <AddPropertyForm propertyId={property._id}/>}
```

Comment:
are we calling `AddPropertyForm` to add a tenant?

i am eventually going to refactor all form components into one single component but i don't think it's applicable as of now? or am i missing something 🤔